src/components/TodoTask.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Please rename this component to be something like TodoList. Since this is a full list of tasks not just one task.

Reply:
Can I change to TodoTasks because we're already have TodoList interface?

Reply:
Yes, sure